src/Compilers/Core/MSBuildTask/Microsoft.Build.Tasks.CodeAnalysis.csproj

```diff
@@ -6,7 +6,7 @@
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis.BuildTasks</RootNamespace>
     <DefaultLanguage>en-US</DefaultLanguage>
-    <TargetFrameworks>netcoreapp3.1;net472</TargetFrameworks>
+    <TargetFrameworks>net6.0;net472</TargetFrameworks>
```

Comment:
❓ Why is this one changing? Do we not support MSBuild prior to net6.0 anymore?

Reply:
At this point we should not be. Our binaries are usually always inserting into .NET SDK Preview and Servicing. At the moment servicing is now the 6.0 SDK hence that should be the lowest TFM that we need to support out of `main`. 

Anyone who is depending on our `main` branch in a `net5.0` world isn't known to us right now.